ulib/LowStar.Monotonic.Buffer.fst

```diff
@@ -1500,41 +1505,76 @@ let mgcmalloc_of_list #a #rrel r init =
   let content: HST.mreference (Seq.lseq a (U32.v len)) (srel_to_lsrel (U32.v len) rrel) =
     HST.ralloc r s
   in
-  let b = Buffer len content 0ul len in
+  let b = Buffer len content 0ul (Ghost.hide len) in
   b
 
-#push-options "--max_fuel 0 --initial_ifuel 1 --max_ifuel 1 --z3rlimit 64"
+#push-options "--max_fuel 0 --initial_ifuel 1 --max_ifuel 1 --z3rlimit 128 --z3cliopt smt.qi.EAGER_THRESHOLD=5"
 let blit #a #rrel1 #rrel2 #rel1 #rel2 src idx_src dst idx_dst len =
   let open HST in
-  match src, dst with
```

Comment:
Hi @R1kM, I modified this proof a bit yesterday so that it works without hints, with lower rlimit and no threshhold options, and has a replayable hint. Just checking if this overwrite here is intentional.

Reply:
Not intentional sorry! Let me restore your proof and update it with the new model